pkg/types/machinepools.go

```diff
@@ -10,6 +10,16 @@ import (
 	"github.com/openshift/installer/pkg/types/vsphere"
 )
 
+// EncryptionStrategy is a strategy for how the OS disks will be encrypted.
+type EncryptionStrategy string
```

Comment:
this has potential to collide with Cloud Based Encryption strategy.. so maybe `OSEncryptionStrategy`

Reply:
Done.